snprc_ehr/src/client/SndEventsWidget/components/EventListingGridPanel.tsx

```diff
@@ -133,7 +148,7 @@ export const EventListingGridPanelImpl: FC<EventListingProps> = memo((props: Eve
     const renderButtons = () => {
         return (
             <div className="manage-buttons">
-                {<Button disabled={true /*subjectIDs.length != 1*/} bsStyle={'success'}
+                {<Button disabled={!hasWritePermission || subjectIDs.length != 1 || subjectIDs[0] == ""} bsStyle={'success'}
```

Comment:
i think here instead of checking the sujbectIds length !=1 , we should check for length > 0 and subjectIds array is not empty

Reply:
No, if it is greater than 1, it should be disabled here. We are only enabling it if one subjectID is selected.

Reply:
got it got it Jay disabling the edit icon if more than 1 subjectId. sorry i miss understood it.